packages/services/storage/migrations/migrate.ts

```diff
@@ -29,17 +29,15 @@ const isDownCommand = cmd === 'down';
 // We are building a "cli" out of the package, so we need a workaround to pass the command to run.
 if (process.env.MIGRATOR === 'up' && !isCreateCommand && !isDownCommand) {
   console.log('Running the UP migrations');
-  migrator
-    .up()
-    .then(() => migrateClickHouse())
-    .then(() => {
-      process.exit(0);
-    })
-    .catch(error => {
-      console.error(error);
-      process.exit(1);
-    });
+
+  try {
+    await migrator.up();
+    await migrateClickHouse();
+  } catch (error) {
+    console.error(error);
+    process.exit(1);
+  }
```

Comment:
`Async/Await` is better, and `top-level await` should also be available, also `process.exit(0)` is redundant and can be removed